src/main/java/org/helpberkeley/memberdata/ControlBlock.java

```diff
@@ -111,18 +113,12 @@ private void auditColumnNames() {
         }
     }
 
-    public static ControlBlock create(String csvData) {
+    public static ControlBlock create(String csvData) throws IOException, CsvException {
 
-        // Normalize lines
-        String normalized = csvData.replaceAll("\\r\\n?", "\n");
-        // Break into lines
-        String[] lines = normalized.split("\n");
-
-        String header = lines[0];
-
-        if (! header.contains(Constants.CSV_SEPARATOR)) {
-            throw new MemberDataException(BAD_HEADER_ROW);
-        }
+        CSVListReader csvReader = new CSVListReader(new StringReader(csvData));
+        List<List<String>> lines = csvReader.readAllToList();
+        List<String> header = lines.get(0);
+        csvReader.close();
 
         String version = new VersionParser(lines).version();
```

Comment:
Changing this to:
```
    public static ControlBlock create(String csvData) {

        List<List<String>> lines;

        try (StringReader reader = new StringReader(csvData)) {
            lines = new CSVListReader(reader).readAllToList();
        }
        catch (IOException |CsvException ex) {
            throw new MemberDataException(ex);
        }

        List<String> header = lines.get(0);
        String version = new VersionParser(lines).version();
```
gets rid of exposing the IOException and CsvException exceptions.  This is a reasonable thing to do because we were already throwing a MemberDataException from several places in this method, and so the calling code has to be able to deal with.  Additionally MemberDataException extends RuntimeException, an unchecked exception.  This means that calling code does not need to catch or declare it.  Thoughout the code you will see MemberDataException thrown at various levels, but generally only caught and dealt with at high levels (e.g. in Main).

Another thing here is that the explicit call to close() was replaced with a [try with resources statement](https://docs.oracle.com/javase/8/docs/technotes/guides/language/try-with-resources.html).